experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -21,22 +21,22 @@ mutable struct LinearlyReductiveGroup
 
     function LinearlyReductiveGroup(sym::Symbol, m::Int, fld::Field) #have not decided the representation yet
         #check char(fld)
-        @assert sym == :SL 
+        @assert sym == :SL && characteristic(fld) == 0
         R, _ = polynomial_ring(fld, :z => (1:m,1:m))
         return LinearlyReductiveGroup(sym, m, R)
     end
 
     function LinearlyReductiveGroup(sym::Symbol, m::Int, pring::MPolyRing) #the ring input is the group ring
         #check char(field)
         G = new()
-        @assert sym == :SL 
-        fld = base_ring(pring)
+	fld = base_ring(pring)
+        @assert sym == :SL  && characteristic(fld) == 0 
         characteristic(fld) == 0 || error("Characteristic should be 0 for linearly reductive groups")
         G.field = fld
         @req m^2  == ngens(pring) "ring not compatible"
         G.group = (sym,m)
         G.reynolds_operator = reynolds_slm
-        M = matrix(pring, m, m, gens(pring))
+        M = transpose(matrix(pring, m, m, gens(pring)))
```

Comment:
That's not a docu change but a change in behavior. Is this covered by a test? What is it fixing?

(Not arguing against this, I have no idea, but I also don't want this to accidentally slip through)

Reply:
This is correct now and fits with what we write in the book.

Reply:
And, it fits with the Disquisitiones Arithmetica.

Reply:
As discussed: I am happy to trust you this is correct as in: fits what we document and fits our book.

That it fits other books or papers only is relevant if we establish that they use exact same conventions as we and hence are supposed to give identical results. I had a look at that a few months ago, but I didn't keep my notes, and don't have time to repeat this right now. Anyway, as I already wrote, I am happy to trust you, I just want it to be explicit and not "by oversight" ;-). 

BTW, if we verified it fits with the convention in those publications, but does not fit with the conventions in Derksen and Kemper; then I think we should actually point that out in the documentation, perhaps even with an example, as this is of course highly relevant for users of our code. And since we refer to e.g. Derksen/Derksen-Kemper in there, this is doubly important as a user might expect the reverse, or not even be aware of the differences in conventions.

(Ideally we would also state how it compares to Singular and Magma conventions, I guess?)